test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests.cs

```diff
@@ -226,7 +226,22 @@ public SamplesEndToEndTests(TestFixture fixture)
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
             string body = await response.Content.ReadAsStringAsync();
             Assert.Equal("text/plain", response.Content.Headers.ContentType.MediaType);
-            Assert.Equal("Hello Amy", body);
+            Assert.Equal("Hello Mathew,Amy", body);
```

Comment:
In v1 when we created the http req object passed to user code, we were deduping req.query. However in v2 we aren't. I think this is OK - as the test below shows both Node and .NET are consistent in that respect.